HelpSource/Classes/Ringz.schelp

```diff
@@ -6,9 +6,8 @@ categories::  UGens>Filters>Linear
 
 Description::
 
-This is the same as  link::Classes/Resonz:: , except that instead of a
-resonance parameter, the bandwidth is specified in a 60dB ring decay
-time. One Ringz is equivalent to one component of the
+This is the same as  link::Classes/Resonz:: , except that it is constant skirt gain filter, meaning that the peak gain depends on the value of Q. Also, instead of the
```

Comment:
I made a suggestion to change the Resonz help file https://github.com/supercollider/supercollider/pull/1509

Resonz is a constant 0 dB gain filter, Ringz is constant skirt, this is the fundamental difference between the two. Other than that, they're the same. In practice, when you change the resonance parameter in Rignz, it'll add more gain to the filter, whereas in Resonz the peak gain will remain at 0 dB. 

Anyway, like I said, I made the changes in Resonz so this is clearer. The way it was before wasnt clear and was actually misleading. It also didn't make sense to me not making the relationship between both of them in the help file of Resonz.